src/toil/statsAndLogging.py

```diff
@@ -112,9 +112,10 @@ def callback(fileHandle):
                 pass
             else:
                 def logWithFormatting(jobStoreID, jobLogs):
-                    logFormat = '\n%s    ' % jobStoreID
                     logger.debug('Received Toil worker log. Disable debug level '
-                                 'logging to hide this output\n%s', logFormat.join(jobLogs))
+                                 'logging to hide this output')
+                    for line in jobLogs:
+                        logger.debug('%s    %s', jobStoreID, line)
```

Comment:
I was worried about the exact same thing as well--but it doesn't seem to happen, at least in my logs. Maybe this is some setting I've changed? I get, for example:

```
Received Toil worker log. Disable debug level logging to hide this output
'CollateBlasts' h/p/jobC9Idbl    ---TOIL WORKER OUTPUT LOG---
'CollateBlasts' h/p/jobC9Idbl    DEBUG:toil.worker:Next available file descriptor: 6
'CollateBlasts' h/p/jobC9Idbl    INFO:toil:Running Toil version 3.7.0a1-8ea561ee1555c297b8e645e4aa34430ffe1efcbd-dirty.
'CollateBlasts' h/p/jobC9Idbl    DEBUG:toil:Configuration: {'rescueJobsFrequency': 3600, 'logLevel': 'DEBUG', 'maxMemory': 9223372036854775807, 'jobStore': 'file:/home/joel/progressiveCactusNHMerge/evolverMammals/jobStore', 'defaultPreemptable': False, 'disableHotDeployment': False, 'maxPreemptableServiceJobs': 9223372036854775807, 'servicePollingInterval': 60, 'workDir': None, 'stats': False, 'disableCaching': False, 'sseKey': None, 'nodeOptions': None, 'environment': {}, 'minNodes': 0, 'cleanWorkDir': 'always', 'maxCores': 9223372036854775807, 'minPreemptableNodes': 0, 'maxPreemptableNodes': 0, 'maxDisk': 9223372036854775807, 'scaleInterval': 30, 'deadlockWait': 2000, 'preemptableNodeType': None, 'nodeType': None, 'clusterStats': None, 'defaultCores': 1, 'parasolMaxBatches': 10000, 'cseKey': None, 'betaInertia': 1.2, 'maxNodes': 10, 'scale': 1, 'writeLogs': None, 'badWorker': 0.0, 'defaultDisk': 2147483648, 'mesosMasterAddress': 'localhost:5050', 'restart': True, 'useAsync': True, 'preemptableCompensation': 0.0, 'parasolCommand': 'parasol', 'workflowID': 'dd43ab96-2268-4dc0-8fbf-9790e3133511', 'alphaPacking': 0.8, 'maxServiceJobs': 9223372036854775807, 'readGlobalFileMutableByDefault': False, 'badWorkerFailInterval': 0.01, 'maxLogFileSize': 64000, 'defaultMemory': 2147483648, 'preemptableNodeOptions': None, 'workflowAttemptNumber': 4, 'maxJobDuration': 9223372036854775807, 'clean': 'onSuccess', 'provisioner': None, 'batchSystem': 'singleMachine', 'retryCount': 0, 'writeLogsGzip': None}
'CollateBlasts' h/p/jobC9Idbl    DEBUG:toil.worker:Parsed jobGraph
```

(There are DEBUG statements, but those are from the worker rather than the leader.) The `TOIL WORKER LOG` header is prefixed in the output, so this is definitely with my changes included.